omnibus/files/private-chef-ctl-commands/rotate_credentials.rb

```diff
@@ -122,6 +137,77 @@
   exit(0)
 end
 
+add_command_under_category "require-credential-rotation", "credential-rotation", "Disable the Chef Server and require credential rotation", 2 do
```

Comment:
Disable the chef server sounds ominous. Is that the right description? Saying Stop the Chef Server is probably better, since that's the actually command being run. Disabling the chef server makes me think I'm about to royally screw something up.